feconf.py

```diff
@@ -68,6 +69,9 @@
 RULES_DESCRIPTIONS_FILE_PATH = os.path.join(
     os.getcwd(), 'extensions', 'interactions', 'rules.json')
 
+# The ID of the classifier to be used as a default string classifier.
+DEFAULT_STRING_CLASSIFIER = 'LDAStringClassifier'
```

Comment:
Btw, tomorrow this constant might point to a different classifier. Would it be wise to name it as CLASSIFIER_ID_LDA_STRING_CLASSIFIER? I think DEFAULT_STRING_CLASSIFIER (or a similar name) makes more sense.